test/end-to-end/test_license.ps1

```diff
@@ -0,0 +1,82 @@
+Remove-Item -Recurse -Force -ErrorAction SilentlyContinue "$HOME/.hab/accepted-licenses"
+Remove-Item -Recurse -Force -ErrorAction SilentlyContinue "/hab/accepted-licenses"
+
+Describe "license" {
+    It "version check without license works" {
+        ./target/debug/hab --version
```

Comment:
I think all of the `./target/debug/` instances were accedentally left here during local testing.

Reply:
Yep :sweat_smile: , thanks! Addressed in [fdf8c8](https://github.com/habitat-sh/habitat/pull/7834/commits/fdf8c8518285352a45d44e5bb0f71578c40f349b)